client/web/web.go

```diff
@@ -674,6 +699,10 @@ func (s *Server) serveGetNodeData(w http.ResponseWriter, r *http.Request) {
 		ACLAllowsAnyIncomingTraffic: s.aclsAllowAccess(filterRules),
 	}
 
+	ipv4, ipv6 := s.selfNodeAddresses(r, st)
+	data.IPv4 = ipv4.String()
```

Comment:
Note that either one could be invalid. If users disable v4 or v6 on their tailnet